src/main/java/emissary/output/DropOffUtil.java

```diff
@@ -995,6 +995,19 @@ public Date parseEventDate(final String dateString, final boolean supplyDefaultO
         return date;
     }
 
+    private static int countInstances(String text, String find) {
```

Comment:
Can we replace with ```org.apache.commons.lang3.StringUtils.countMatches``` instead of creating a custom method?

Reply:
The implementations are basically the same, so the change has been made.